kuzzleio/sdk-go
# Wrap Server controller for Android SDK

## What does this PR do ?

Wrap Server controller methods for Android SDK in order to take and return JSONObject type when needed.

### How should this be manually tested?

run docker run --rm -it -v "$(pwd)":/go/src/github.com/kuzzleio/sdk-go android-x86 /build.sh
Use internal/wrappers/build/android/app/build/outputs/aar/app-release.aar in an android project and see if you have JSONObject type on the Server controller's methods

## Other changes

I had to move the typemap concerning the `now()` method in this new typemap because swig only take the last typemap of type javacode for a given class. Since there were only the typemap for now() in typemap.i I deleted it.
